tests/unit/cli/admin/test_import_targets.py

```diff
@@ -98,25 +123,36 @@ def test__import_csv_to_rstuf(self):
 
         result = import_targets._import_csv_to_rstuf(
             fake_db_client,
-            fake_rstuf_table,
+            fake_rstuf_files,
+            "fake_rstuf_roles",
             ["csv1", "csv2"],
             "fake_succinct_roles",
         )
 
         assert result is None
         assert import_targets._parse_csv_data.calls == [
-            pretend.call("csv1", "fake_succinct_roles"),
-            pretend.call("csv2", "fake_succinct_roles"),
+            pretend.call(
+                fake_db_client,
+                "fake_rstuf_roles",
+                "fake_succinct_roles",
+                "csv1",
+            ),
+            pretend.call(
+                fake_db_client,
+                "fake_rstuf_roles",
+                "fake_succinct_roles",
+                "csv2",
+            ),
         ]
         assert fake_db_client.execute.calls == [
             pretend.call(
-                fake_rstuf_table.insert(), [{"k1": "v1", "k2": "v2"}]
+                fake_rstuf_files.insert(), [{"k1": "v1", "k2": "v2"}]
             ),
             pretend.call(
-                fake_rstuf_table.insert(), [{"k1": "v1", "k2": "v2"}]
+                fake_rstuf_files.insert(), [{"k1": "v1", "k2": "v2"}]
             ),
         ]
-        assert fake_rstuf_table.insert.calls == [
+        assert fake_rstuf_files.insert.calls == [
             pretend.call(),
```

Comment:
Any idea why `fake_rstuf_files.insert.calls` calls are three when there are two `csv` files?

Reply:
I am fixing it.
It is because I was also calling it on lines 149 and 152.